src/components/Community/data.json

```diff
@@ -28,13 +28,13 @@
   },
   "documentation": [
     {
-      "url": "/doc/get-started",
+      "url": "/doc/tutorials/get-started",
       "title": "Get started",
       "description": "Reproducible walkthroughs on advanced topics"
     },
     {
       "url": "/doc/tutorials",
-      "title": "Tutorials",
+      "title": "DVC Tutorials",
```

Comment:
why DVC Tutorials?

Reply:
I was changing just back from "Learn DVC", didn't remember it was just "Tutorials" before. Reverted.